scripts/zones/Kuftal_Tunnel/mobs/Amemet.lua

```diff
@@ -1,37 +1,199 @@
-    {x = 7.12, y = 0.02, z = 117.76},
-    {x = 4.74, y = 0.27, z = 99.64},
-    {x = 17.30, y = -4.22, z = 54.34},
-    {x = 19.45, y = -8.75, z = 40.65},
-    {x = 25.23, y = -8.96, z = 24.95},
-    {x = 40.71, y = -8.75, z = 21.33},
-    {x = 64.58, y = -0.87, z = 13.55},
-    {x = 88.68, y = -0.44, z = 12.39},
+    { x = 102.61, y = 0.92, z = 1.60 },
+}
+
+local pathA =
+{
+    { x = 102.61, y = 0.92, z = 1.60 },
+    { x = 90.06, y = -0.28, z = 12.24 },
+    { x = 67.60, y = -0.34, z = 12.42 },
+    { x = 54.61, y = -4.11, z = 16.89 },
+    { x = 25.03, y =  -9.28, z = 23.12 },
+    { x = 19.43, y = -7.95, z = 44.37 },
+    { x = 18.80, y = -6.63, z = 47.77 },
+    { x = 17.72, y = -5.14, z = 51.88 },
+    { x = 16.93, y = -3.95, z = 55.33 },
+    { x = 16.33, y = -3.09, z = 57.46 },
+    { x = 14.76, y = -1.78, z = 60.84 },
+    { x = 11.86, y = -0.60, z = 65.58 },
+    { x = -0.52, y = 0.00, z = 82.02 },
+}
+
+local pathAb =
+{
+    { x = 3.06, y = 0.15, z = 125.56 },
+    { x = 40.12, y = 0.000, z = 158.89 },
+    { x = 60.35, y = -4.24, z = 143.23 },
+    { x = 79.86, y = -8.75, z = 139.89 },
+    { x = 85.86, y = -8.82, z = 138.11 },
+    { x = 91.87, y = -8.83, z = 142.52 },
+    { x = 101.60, y = -8.71, z = 143.17 },
+    { x = 100.62, y = -8.75, z = 78.86 },
+    { x = 108.29, y = -4.60, z = 54.26 },
+    { x = 118.98, y = 0.00, z = 38.12 },
+    { x = 102.61, y = 0.92, z = 1.60 },
+}
+
+local pathB =
+{
+    { x = 102.61, y = 0.92, z = 1.60 },
+    { x = 109.49, y = 0.59, z = -0.07 },
+    { x = 124.47, y = 0.00, z = -42.08 },
+    { x = 91.93, y = -5.15, z = -58.24 },
+    { x = 66.46, y = -8.81, z = -62.91 },
+    { x = 60.86, y = -9.02, z = -73.80 },
+    { x = 57.09, y = -9.37, z = -96.88 },
+    { x = 38.39, y = -8.62, z = -98.74 },
+    { x = 15.73, y = -1.02, z = -94.88 },
+    { x = -0.65, y = 0.00, z = -84.13 },
+    { x = -40.24, y = 0.00, z = -38.35 },
+    { x = -44.36, y = 0.68, z = 13.98 },
+    { x = -31.28, y = -0.23, z = 48.64 },
+    { x = -0.52, y = 0.00, z = 82.02 },
+}
+
+local pathBb =
+{
+    { x = 3.06, y = 0.15, z = 125.56 },
+    { x = 40.12, y = 0.000, z = 158.89 },
+    { x = 60.35, y = -4.24, z = 143.23 },
+    { x = 79.86, y = -8.75, z = 139.89 },
+    { x = 85.86, y = -8.82, z = 138.11 },
+    { x = 91.87, y = -8.83, z = 142.52 },
+    { x = 101.60, y = -8.71, z = 143.17 },
+    { x = 100.62, y = -8.75, z = 78.86 },
+    { x = 108.29, y = -4.60, z = 54.26 },
+    { x = 118.98, y = 0.00, z = 38.12 },
+    { x = 102.61, y = 0.92, z = 1.60 },
+}
+
+local pathFind =
+{
+    ['pathFind1'] = function(mob, reversePath)
+        if reversePath == 0 or reversePath == 1 then
+            local pathRnd = math.random(0,1)
+            local reverseCheck = math.random(0,2)
+            if pathRnd == 1 then
+                mob:setLocalVar("mobPath", 2)
+                if reverseCheck ~= 2 then
+                    mob:setLocalVar("reversePath", 0)
+                    path = pathA
```

Comment:
Typically for any variable, you'll want to instantiate it before use as a local var. 

In all of these blocks you're using the `path` variable, you'll want to have a line with `local path = {}` at the start since it ends up being a table. So for instance in this block I'm commenting on, you'd want that line just below the `reverseCheck` var.

Reply:
Updated!